lib/components/rtsp-session/rtspsession-component.js

```diff
@@ -157,7 +161,23 @@ class RtspSessionComponent extends Component {
     if (!this._sessionId && !ended) {
       // Response on first SETUP
       this._sessionId = Rtsp.sessionId(msg.data)
+      const sessionTimeout = Rtsp.sessionTimeout(msg.data)
+      if (sessionTimeout !== null) {
+        // The server specified that sessions will timeout if not renewed.
+        // In order to keep it alive we need periodically send a RTSP_OPTIONS message
+        if (this._renewSessionInterval) {
```

Comment:
`if (this._renewSessionInterval !== null)`, I'm not sure about the IDs but it's a positive integer, so I guess it could theoretically be 0.